server.js

```diff
@@ -1,15 +1,43 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const os = require('os');
+const clc = require('cli-color');
+const crypto = require('crypto');
 
-const app = express();
-app.use(bodyParser.json({ extended: true }));
+const SIGNATURE_SECRET = 'my_secret';
```

Comment:
Can we use dotenv to handle this? I do a similar thing with the fivetran-connect-card-script-nodejs that makes it a bit easier to catch and customize without feeling like you're committing your secret to the repo: https://github.com/fivetran-jimmyhooker/fivetran-connect-card-script-nodejs